README.md

```diff
@@ -1,72 +1,67 @@
+# sul-requests
 [![Build Status](https://travis-ci.org/sul-dlss/sul-requests.svg?branch=master)](https://travis-ci.org/sul-dlss/sul-requests)
-[![Code Climate](https://codeclimate.com/github/sul-dlss/sul-requests/badges/gpa.svg)](https://codeclimate.com/github/sul-dlss/sul-requests)
-[![Code Climate Test Coverage](https://codeclimate.com/github/sul-dlss/sul-requests/badges/coverage.svg)](https://codeclimate.com/github/sul-dlss/sul-requests/coverage)
-[![Latest Tagged version](https://badge.fury.io/gh/sul-dlss%2Fsul-requests.svg)](https://badge.fury.io/gh/sul-dlss%2Fsul-requests)
-
-
-# SUL Requests
-
-SUL Requests in a rails application that allows users to request materials from the Stanford University Library.  This application aims to:
-
-1. Support Scan & Deliver service for limited materials.
-2. Provide better support for the Special Collections request process. *Request all materials from a single online form, regardless of home location.*
-3. Address known usability/accessibility issues with existing request forms.
-4. Support necessary administrative tasks, including request mediation, updating the paging schedule, adding/removing requestable item categories.
-
-
-## Requirements
-
-1. Ruby (2.5.3 or greater)
-2. Rails (5.2.2 or greater)
-3. A database
-
-## Installation
-
-Clone the repository
-
-    $ git clone git@github.com:sul-dlss/sul-requests.git
-
-Change directories into the app and install dependencies
-
-    $ bundle install
-
-Run database migrations
-
-    $ rake db:migrate
-
-Start the development server
-
-    $ rails s
-
-## Configuring
-
-Configuration is handled through the [RailsConfig](/railsconfig/rails_config) settings.yml files.
-
-### WorkGroups
+![tested on ruby 3.0](https://img.shields.io/badge/ruby-v3.0-red)
+[![Current release](https://img.shields.io/github/v/release/sul-dlss/sul-requests)](https://github.com/sul-dlss/sul-requests/releases)
+[![License](https://img.shields.io/badge/license-apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
+<p align="center">a rails application for requesting materials from the Stanford University Library.</p>
+<img src="preview.png" align="center" alt="request form for paging an item from off-site storage to green library">
+
+## features
+- request all materials from a single online form, regardless of home location.
+- support Scan & Deliver service for limited materials.
+- provide support for the Special Collections request process.
+- avoid known usability/accessibility issues with third-party request forms.
+- support administrative tasks, including request mediation, updating the paging schedule, and adding/removing requestable item categories.
+## local development
+### requirements
+- Ruby (tested on 3.0)
+- Rails (tested on 6.1)
+- a database
```

Comment:
@thatbudakguy this feels mysterious, could you say something like "a database (we have developed locally with xxx)"

Reply:
just checked and I think it's always sqlite, even in prod... @cbeer can you confirm?